otseobande/maze-problem
# #162250650 Add game play icons to game grid

**What does this PR do?**

This PR implements game play icon placement on game grid

**Description of Task to be completed?**

- add game icons to assets/images folder
- update cell to show game icons
- update game board to position icons
- write tests

**How should this be manually tested?**

- Fetch and check out to the associated branch
- Start the development server by running `$ yarn start`
- Click the `Start Game` button
- You should see a grid with mushrooms and a mario

**What are the relevant pivotal tracker stories?**

- [#162250650](https://www.pivotaltracker.com/story/show/162250650)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-11-28 at 4 04 59 am" src="https://user-images.githubusercontent.com/17123317/49126136-d1121300-f2c2-11e8-9885-51220815bd7a.png">
